CRM/Financial/Form/PaymentEdit.php

```diff
@@ -67,11 +67,7 @@ public function preProcess() {
    */
   public function setDefaultValues() {
     $defaults = $this->_values;
-    // Format money fields - localize for display
-    $moneyFields = ['total_amount', 'fee_amount', 'net_amount'];
-    foreach ($moneyFields as $field) {
-      $defaults[$field] = CRM_Utils_Money::formatLocaleNumericRoundedForDefaultCurrency($this->_values[$field]);
-    }
+    $defaults['total_amount'] = CRM_Utils_Money::formatLocaleNumericRoundedForDefaultCurrency($this->_values['total_amount']);
```

Comment:
@eileenmcnaughton is the gist that fee_amount and net_amount aren't exposed on this form or something?

Reply:
@seamuslee001 yep